messaging/usecase/student_usecase.go

```diff
@@ -203,26 +107,31 @@ func (s studentUseCase) ListenStudentDelete() {
 	failOnError(err, failQueue)
 
 	err = ch.QueueBind(
-		q.Name,       // queue name
-		"profile.deleted",            // routing key
-		"profile", // exchange
+		q.Name,            // queue name
+		"profile.updated", // routing key
```

Comment:
there should be one for created, updated, deleted, please refer to the original code and add them back